docs/targets/awskinesis.md

```diff
@@ -20,31 +20,31 @@ Open the Bridge creation screen and add a Target of type `AWS Kinesis`.
 
 ![Adding a Kinesis Target](../images/aws-targets/aws-kinesis-bridge-create-1.png)
 
-In the Target creation form, provide a name for the event Target, and add the following information:
+In the Target creation form, provide a name for the event Target and add the following information:
 
-* **AWS Secret**: Reference a [TriggerMesh secret](../guides/secrets.md) containing an AWS API key and Secret as discussed in the prerequisites
-* **AWS ARN**: The ARN that points to the AWS Kinesis stream
-* **Partition**: The Kinesis partition to publish the events to
+* **AWS Secret**: Reference a [TriggerMesh secret](../guides/secrets.md) containing an AWS API key and Secret as discussed in the prerequisites.
+* **AWS ARN**: The ARN that points to the AWS Kinesis stream.
+* **Partition**: The Kinesis partition to publish the events to.
 
 There is an optional toggle flag indicating if the full CloudEvent should be sent
-to Kinesis. By default, this is disabled which means the event payload
-will be sent only.
+to Kinesis. By default, this is disabled which means only the event payload
+will be sent.
 
 ![AWS Kinesis Target form](../images/aws-targets/aws-kinesis-bridge-create-2.png)
 
 After clicking the `Save` button, the console will self-navigate to the Bridge editor. Proceed by adding the remaining components to the Bridge.
 
 ![Bridge overview](../images/aws-targets/aws-kinesis-bridge-create-3.png)
 
-After submitting the bridge, and allowing some configuration time, a green check mark on the main _Bridges_ page indicates that the bridge with the AWS Kinesis Target was successfully created.
+After submitting the Bridge, and allowing for some configuration time, a green check mark on the main _Bridges_ page indicates that the Bridge with the AWS Kinesis Target was successfully created.
```

Comment:
@odacremolbap and @antoineco I like how @alicenstar added a capital letter for Bridge. I assume this was to stand out to the user. Maybe we could highlight them. What do you all think?

Reply:
Agree, since "bridge" is a common word, the capital letter makes it stand out. I like it.

Reply:
Awesome. In the rules guide Thomas is making, Thomas is also going to add this in the rules.

Reply:
Glad you guys liked that - I figured it would be helpful since your docs already highlight other pertinent words such as Target.